template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/BlackListedTimeBasedAutoscaleRole.java

```diff
@@ -10,7 +10,6 @@ public enum BlackListedTimeBasedAutoscaleRole {
     SERVER,
     NIFI_NODE,
     NAMENODE,
-    IMPALAD,
```

Comment:
Does impala re-use the IMPALAD role for the co-ordinators and executors ? This particular change is probably required, but we may need to add additional checks to ensure that autoscaling is only enabled for nodes running with the executor specialization. That will be worth exploring.

See the DataMart template. Example: 
`"roleConfigGroups": [
        {
          "refName": "impala-IMPALAD-COORDINATOR",
          "roleType": "IMPALAD",
          "configs": [
            {
              "name": "impalad_specialization",
              "value": "COORDINATOR_ONLY"
            },`

Reply:
Yes sid,
As we have removed IMPALAD from blacklisted list, now we can scale up executor as well as coordinator.
For coordinator, we can add additional checks for checking it out and stopping the customers from doing that but in the UI as it takes recommodation from CB both executor and coordinator will be visible there.

Reply:
The API on the CB side then needs to be enhanced to ensure it only allows for EXECUTORS in case of IMPALA ? (Presumably via the `impalad_specialization` configuration property being set to "COORDINATOR_ONLY")

(Similar check on the Periscope side as well when autoscaling is enabled or disabled - so utility method)